tests/e2e/models/lfm2_moe/e2e_plugins/runners/text_generation.py

```
@@ -0,0 +1,321 @@
+                "steps": len(logits),
+                "prompt_token_ids": input_ids,
+            }}))
+            """
+        )
+
+        command = [ctx.runtime_python_path() or sys.executable, "-c", script]
+        env = dict(os.environ)
+        if ctx.ld_library_path:
+            env["LD_LIBRARY_PATH"] = ctx.ld_library_path
+        started = time.monotonic()
+        result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)